html/tests/masthead.tests.js

```diff
@@ -371,28 +371,27 @@ describe('masthead UI Events tests', () => {
     );
   });
 
-  it(`should add checkScrollDirection event 
+  it(`should add checkScrollDirection event
       listener if menu is visible`, () => {
     const attached = toggleScrollEvent(true);
     expect(attached).toBe(true);
   });
 
-  it(`should not add checkScrollDirection event listener 
+  it(`should not add checkScrollDirection event listener
       if menu is not visible`, () => {
     const attached = toggleScrollEvent(false);
     expect(attached).toBe(false);
   });
 
-  it(`should checkScrollDirection on resize and 
+  it(`should checkScrollDirection on resize and
       return true if menu visible`, () => {
-    iconContainerDiv.setAttribute('style', 'display: none');
```

Comment:
Thank you for updating this test!

Reply:
This was @RCopeland  but I will happily take credit for his hard work